_posts/2015-05-22-authentication-end-point.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: Authentication end-point
+status: public
+type: post
+published: true
+author: Ken Mugrage
+---
+
+Starting 15.2.0 Go Server will expose authentication end-point. What this means is Go users can add "custom" authentication schemes through plugins. With [plugin settings](http://www.go.cd/documentation/developer/writing_go_plugins/plugin_settings/plugin_settings_overview.html) & [web request handling ability](http://www.go.cd/documentation/developer/writing_go_plugins/handling_web_requests.html) plugin developers get enough flexibility to write any authentication plugin they intend to write.
```

Comment:
"Go Server will expose authentication end-point"

to

"Go Server will expose authentication an end-point"

"What this means is Go users can add "custom" authentication schemes through plugins. "
to
"What this means is Go plugin authors can add support for "custom" authentication schemes via plugins, allowing users to use these plugins on a GO server instance to use an alternate authentication scheme."

Not required.
"plugin developers get enough flexibility to write any authentication plugin they intend to write."